README.md

```diff
@@ -6,6 +6,11 @@ Some reusable Terraform modules.
 
 This module sets up a VPC, and a Consul and Nomad cluster to allow you to run applications on.
 
+## [Amazon Linux](modules/amazon-linux)
```

Comment:
This doesn't exist any more right?

Reply:
Yup sorry missed this out, will remove and add a section about `Roles`